crates/circuits/primitives/src/range_tuple/mod.rs

```diff
@@ -100,6 +104,9 @@ pub struct RangeTupleCheckerChip<const N: usize> {
     count: Vec<Arc<AtomicU32>>,
 }
 
+#[derive(Debug, Clone)]
+pub struct SharedRangeTupleCheckerChip<const N: usize>(Arc<RangeTupleCheckerChip<N>>);
```

Comment:
did you make a newtype for protected type purposes or because of some practical reason around traits?